app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1533,6 +1533,22 @@ protected void onResume() {
     }
 
 
+    public String getDefaultLocale(){
+
+        String language;
+        Locale defaultLocale = Locale.getDefault();
+        if(defaultLocale.toString().length()>2)
+            language = defaultLocale.toString().substring(0, Math.min(defaultLocale.toString().length(), 2));
```

Comment:
No need to hack your way around this.
Android already allows you to get language code in different formats.

http://stackoverflow.com/a/23168383/5471095